### DISCUSSION TITLE
Can't install Yi Hack on Yi Dome Camera U (2K)

### DISCUSSION THREAD

**Discussion by marcokldm02** [roleoroleo/yi-hack-Allwinner-v2#928] [Q&A]
Hi, I followed all the steps to install Yi Hack Allwinner v2 on my Yi Dome Camera U (2K), but it seems that the camera does not want to recognize it. When I insert the SD card the camera turns on normally with the original firmware. I also tried to backup the original firmware and it does not work.

Can anyone help me? Thanks

**Comment by pitisec** [comment]
Hi, be sure to format sd card with fat32 format. 
I have exact model and for me worked great. 
(prefix is h51ga)
